graylog2-web-interface/src/views/components/common/EditableTitle.jsx

```diff
@@ -1,9 +1,32 @@
 // @flow strict
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
 
 import styles from './EditableTitle.css';
 
+const StyledStaticSpan = styled.span(({ theme }) => css`
+  border: 1px solid ${theme.colors.global.contentBackground};
+  padding: 2px 3px;
+  font-size: ${theme.fonts.size.large};
+  display: inline-block;
+`);
+
+const StyledInput = styled.input(({ theme }) => css`
+  border: 1px solid ${theme.colors.input.border};
+  background-color: ${theme.colors.input.background};
+  color: ${theme.colors.input.color};
+  border-radius: 4px;
+  padding: 2px 3px;
+  font-size: ${theme.fonts.size.large};
+  
+  :focus {
+    border-color: ${theme.colors.input.borderFocus};
+    //box-shadow: ${theme.colors.input.boxShadow};
```

Comment:
What is this for? If it is a comment, can we remove it? I am asking, because CSS actually expects comments to be `/* */` (not saying that browsers would not parse it anyway :trollface:).